### DISCUSSION TITLE
Peptide in PK-SIM

### DISCUSSION THREAD

**Discussion by Boulenc** [Open-Systems-Pharmacology/Forum#785] [Questions & Problems] [answered]
Dear community,
I am looking for examples of PBPK modelling in PK-SIM with peptide. With ‘’peptide’’, I mean compound of around 1-2 kDa as MW, without Fc part. I am interested in tissue distribution of such modality. Any literature available? 
Thank you for your help.
Xavier

**Comment by StephanSchaller** [comment, marked as answer]
Dear Xavier,

the [publication on the protein (2-pore) model from PK-Sim](https://link.springer.com/article/10.1007%2Fs10928-017-9559-4) also covers a peptide (inulin, no Fc binding) with around 5.5 kDa (Tissue distribution in Figure 12).
From the top of my head, I don't know of any publications on a 1-2 kDa compound where tissue data are available.

Best,
Stephan

**Comment by Boulenc** [comment]
Thank you Stephan, 
Indeed, Inulin (polysacharide) with small MW, without Fc part, and thereby rapid elimination and diffusion, can be considered to have a behaviour close to a peptide I guess.
Best Regards.
Xavier.
